Modules/exportDKP.lua

```diff
@@ -77,16 +78,15 @@ local function GenerateDKPTables(table, format)
 				numrows = #MonDKP_DKPHistory
 			end
 
-			ExportString = "Listed As: PlayerString(separated by -),dkp,date(epoch),reason,...\n"
-			ExportString = ExportString.."DKPHistory = ";
+			Headers = "player,DKP,date,reason\n"
+			ExportString = Headers.."";

```

Comment:
>  simon-hirst1 2 minutes ago

What is the purpose of this?

Reply:
Was done in a similar way previously, so I changed the variable to be more descriptive and kept it in line with the old style.

Reply:
Claiming to do something "because that's the style" when it does nothing functionally is not a valid reason.

Reply:
If you keep the list of names separated by commas how would you tell where the list of names ends and the rest of the entry begins?

Reply:
They're surrounded by double quotes, doing this allows commas inside the CSV values. 

> Fields containing line breaks (CRLF), double quotes, and commas should be enclosed in double-quotes.

Reply:
I was using the dashes to make it easier on the end user utilizing the CSV, not so much for the addon. Utilizing the dashes it would consider the entire player string one value that could then be broken up and manipulated as needed by the program utilizing it. Keeping commas in there simply means more work for the 3rd party application manipulating the data to figure out if the next value is still part of the player list or if it's the remainder of the entry.

Reply:
Have you had the chance to review this yet? In the current export CSV implementation there are multiple errors that make the CSV output almost unusable for anything unless manually corrected. Can message you on Discord as well if that would be more convenient.